components/avatar/Avatar.js

```diff
@@ -0,0 +1,52 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class Avatar extends PureComponent {
+  static propTypes = {
+    borderColor: PropTypes.oneOf([ 'neutral', 'mint', 'aqua', 'violet', 'teal', 'gold', 'ruby', 'tealdark' ]),
```

Comment:
In most component you use a separate 'dark' prop to create a darker version of some colors. Here you use 'tealdark`. Maybe it's better to keep all components consistent?